src/Sylius/Bundle/ResourceBundle/Routing/ResourceLoader.php

```diff
@@ -169,6 +169,9 @@ private function createRoute(MetadataInterface $metadata, array $configuration,
         if (!empty($configuration['criteria'])) {
             $defaults['_sylius']['criteria'] = $configuration['criteria'];
         }
+        if (!empty($configuration['filterable'])) {
```

Comment:
I used `empty` to avoid adding `filterable: false` to every route specification as that's the default value anyways. I can change it, will take a moment to fix all tests though.